lib/ex_zample/dsl.ex

```diff
@@ -150,7 +150,7 @@ defmodule ExZample.DSL do
       end)
 
     file_name = Macro.underscore(caller.module) <> ".ex_zample_manifest.elixir"
-    file = Path.join(Mix.Project.manifest_path(), file_name)
+    file = Path.join(Mix.Project.build_path(), file_name)
```

Comment:
The `build_path` goes to `_build/[env]/`, in other words to the root folder of users build path, what do you think in joining the `"/ex_zample/manifest/"` for manifest files goes to scoped folder?

Reply:
Make sense to me. Do you mean like this:

```Elixir
Path.join(Mix.Project.build_path() <> "/ex_zample/manifest/", file_name)
```

Reply:
you can also use lists:

```elixir
Path.join([Mix.Project.build_path(), "/ex_zample/manifest/", file_name])
```